superset-frontend/src/dashboard/util/getDashboardUrl.ts

```diff
@@ -18,12 +18,13 @@
  */
 import { URL_PARAMS } from 'src/constants';
 import serializeActiveFilterValues from './serializeActiveFilterValues';
+import { DashboardStandaloneMode } from './constants';
 
 export default function getDashboardUrl(
   pathname: string,
   filters = {},
   hash = '',
-  standalone?: number | null,
+  standalone?: boolean,
```

Comment:
`standalone` can actually be called with the value `2`, in which case it removes the title:
![image](https://user-images.githubusercontent.com/33317356/120749629-9598d500-c50d-11eb-9ffa-97a19968956c.png)
So perhaps `number | boolean` could be a good solution here, as it appears to work as expected with values `undefined`, `true`, `false`, `0`, `1` and `2`?

Reply:
> `standalone` can actually be called with the value `2`, in which case it removes the title:
> ![image](https://user-images.githubusercontent.com/33317356/120749629-9598d500-c50d-11eb-9ffa-97a19968956c.png)
> So perhaps `number | boolean` could be a good solution here, as it appears to work as expected with values `undefined`, `true`, `false`, `0`, `1` and `2`?

I know this, but I don't think it's a good design. When we remove the header there is no chance to return it.

Reply:
I'm fine with that approach - I think this was discussed to some degree on the PR that introduced it: #12918

Reply:
Thanks for your point out.